README.rst

```diff
@@ -22,29 +22,18 @@ via `ODK Aggregate <https://docs.opendatakit.org/aggregate-intro/>`_ or `ODK Bri
 
 Requirements
 ~~~~~~~~~~~~
-To compile the source code of SmartVA-Analyze, the following applications must be installed:
+To compile the source code of SmartVA-Analyze, one the following applications must be installed:
 
 * `Anaconda <https://www.anaconda.com/download/>`_
+* `Miniforge <https://conda-forge.org/miniforge/>`_​
 
-* `Docker <https://www.docker.com/get-started>`_
-
-
-Build Instructions
```

Comment:
I'd change the build instructions rather than deleting them, to something along the lines of:
```
activate the conda environment and then run `./pkg/run-pyinstaller.sh`
```
along with a reference to using [Git Bash](https://gitforwindows.org/) rather than WSL for the windows builds